lib/PTML/Doxygen.cpp

```diff
@@ -585,8 +585,11 @@ std::string ptml::functionComment(const ::ptml::MarkupBuilder &PTML,
   return Builder.emit(std::move(Result));
 }
 
+static Logger ImprecisePositionWarning("imprecise-comment-position-warnings");
```

Comment:
Too specific, I'd just go for `comment-emission`

Reply:
I went with `statement-comment-emission`.